base/collection.go

```diff
@@ -31,6 +31,11 @@ var ErrCollectionsUnsupported = errors.New("collections not supported")
 var _ sgbucket.KVStore = &Collection{}
 var _ CouchbaseStore = &Collection{}
 
+type Keyspace struct {
```

Comment:
Keyspace is probably more correct as bucketname/scope name/collection name.  It doesn't look like this is used in this PR, maybe remove until it's used, so that there's a better context to decide the best structure?

Reply:
This is now unused, removed.